src/doc/rustc/src/targets/known-issues.md

```diff
@@ -0,0 +1,13 @@
+# Known Issues
+This section informs you about known "gotchas". Keep in mind, that this section is (and always will be) incomplete. For suggestions and amendments, feel free to [contribute](https://doc.rust-lang.org/rustc/contributing.html) to this guide.
+
+## Target Features
+Most target-feature problems arise, when mixing code that have the target-feature _enabled_ with code that have it _disabled_. If you want to avoid undefined behavior, it is recommended to build _all code_ (including the standard library and imported crates) with a common set of target-features.
+
+By default, compiling your code with the `-C target-feature` flag will not recompile the entire standard library and/or imported crates with matching target features. Therefore, target features are generally considered as unsafe. Using `#[target_feature]` on individual functions makes the function unsafe.
+
+Examples:
+
+| Target-Feature | Issue | Seen on | Description | Details |
+| -------------- | ----- | ------- | ----------- | ------- |
+| `-C target-feature=+soft-float` | Segfaults and ABI mismatches | `x86 and x86-64` | Using soft-floats disables usage of ```xmm``` registers, but parts of ```std``` are compiled without soft-floats and expect parameters to be passed in ```xmm``` registers. This leads to ABI mismatches. | [#63466](https://github.com/rust-lang/rust/issues/63466) |
```

Comment:
`-C target-feature=-sse` should also cause the same issue.

Reply:
Done, I added `-sse` to the list.